app/views/conditions/dashboard.erb

```diff
@@ -0,0 +1,219 @@
+<div class="header-image jumbotron">
+  <h1>Conditions Overview</h1>
+  <p><a class="btn btn-primary" href="/conditions">All Conditions</a></p>
+</div>
+
+<div class="row">
+  <div class="col-md-offset-2 col-sm-6 col-md-4">
+    <div class="thumbnail">
+      <img class ="img-responsive" src="/images/sunny.jpg" alt="Sunny">
+      <div class="caption">
+        <h3>Conditions for Highest Ride Count</h3>
+        <h4><%= DateFormat.parse(@highest_rides.date) %></h4>
+        <ul>
+          <li>High Temp: <%= @highest_rides.max_temperature_f %></li>
+          <li>Low Temp: <%= @highest_rides.min_temperature_f %></li>
+          <li>Mean Visibility: <%= @highest_rides.mean_visibility_miles %> Miles</li>
+          <li>Humidity: <%= @highest_rides.mean_humidity %></li>
+          <li>Precipitataion: <%= @highest_rides.precipitation_inches %> Inches</li>
+          <li>Wind Speed: <%= @highest_rides.mean_wind_speed_mph %> MPH</li>
+        </ul>
+        <p><a href="/conditions/<%= @highest_rides.id %>" class="btn btn-primary" role="button">Show</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="col-sm-6 col-md-4">
+    <div class="thumbnail">
+      <img class ="img-responsive" src="/images/rain.png" alt="Cloudy">
+      <div class="caption">
+        <h3>Conditions for Lowest Ride Count</h3>
+        <h4><%= DateFormat.parse(@lowest_rides.date) %></h4>
+        <ul>
+          <li>High Temp: <%= @lowest_rides.max_temperature_f %></li>
+          <li>Low Temp: <%= @lowest_rides.min_temperature_f %></li>
+          <li>Mean Visibility: <%= @lowest_rides.mean_visibility_miles %> Miles</li>
+          <li>Humidity: <%= @lowest_rides.mean_humidity %></li>
+          <li>Precipitataion: <%= @lowest_rides.precipitation_inches %> Inches</li>
+          <li>Wind Speed: <%= @lowest_rides.mean_wind_speed_mph %> MPH</li>
+        </ul>
+        <p><a href="/conditions/<%= @lowest_rides.id %>" class="btn btn-primary" role="button">Show</a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<h2>Rides by Maximum Temperature</h2>
+<table class="table table-striped">
+  <tr>
+    <th>Temperature Range</th>
+    <th>Average Number of Rides</th>
+    <th>Highest Number of Rides</th>
+    <th>Lowest Number of Rides</th>
+  </tr>
+  <tr>
+    <td>41-50</td>
+    <td><%= Condition.average_number_of_rides_in_range_max_temp(41..50) %></td>
```

Comment:
we talked about this in your eval, but i'll reiterate it here. currently, we're overstepping the controller's responsibilities. the controller should grab this data from your models and the view should simply display the data. as this view template stands, there's too much logic here because it's actually querying your database by accessing a model directly.